projects/route-analyzer/src/lib/constants.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+export const ANGULAR_CORE = {
+    ANGULAR_ROUTER: '@angular/router',
+    FOR_CHILD: 'forChild',
+    FOR_ROOT: 'forRoot',
+    ROUTER_MODULE: 'RouterModule',
+};
+
+export const ROUTE = {
```

Comment:
Unclear what this is? The names of fields in `AppRoute`? Does it belong with that file instead?

Reply:
Not the name of fields in AppRoute, but rather the name of fields of angular Route.
I renamed it to ROUTE_PROP and in the code, it is used like:
  if (propName === ROUTE_PROP.PATH)